status/flowcells.py

```diff
@@ -178,3 +183,101 @@ def lane_q30(self, flowcell):
             lane_q30[row.key[2]] = row.value["sum"] / row.value["count"]
 
         return lane_q30
+
+class FlowcellNotesDataHandler(SafeHandler):
```

Comment:
Is it possible to abstract common functionality from the Flowcell/workset/project runningnotes classes to superclass? 
I have reviewed this code and don't really have much else to say.